ops/terraform/services/base/values/ephemeral.yaml

```diff
@@ -1,97 +1,65 @@
-      # peerings to ephemeral environments
-      lb_vpc_peerings_json: >-
-        []
+/bfd/${env}/common/nonsensitive/env_name_std: ${env}
+/bfd/${env}/common/nonsensitive/new_relic_metric_host: https://gov-metric-api.newrelic.com
+/bfd/${env}/common/nonsensitive/new_relic_metric_path: /metric/v1
+/bfd/${env}/common/nonsensitive/rds_backup_retention_period: 1
+/bfd/${env}/common/nonsensitive/rds_cluster_identifier: bfd-${env}-aurora-cluster
+/bfd/${env}/common/nonsensitive/rds_iam_database_authentication_enabled: true
+/bfd/${env}/common/nonsensitive/rds_instance_count: 1
+/bfd/${env}/migrator/nonsensitive/instance_type: m6a.large
+/bfd/${env}/migrator/nonsensitive/volume_iops: 3000
+/bfd/${env}/migrator/nonsensitive/volume_size: 100
+/bfd/${env}/migrator/nonsensitive/volume_throughput: 125
+/bfd/${env}/migrator/nonsensitive/volume_type: gp3
+/bfd/${env}/migrator/nonsensitive/sqs_queue_name: bfd-${env}-migrator
+/bfd/${env}/migrator/nonsensitive/new_relic_metric_host: https://gov-metric-api.newrelic.com
+/bfd/${env}/migrator/nonsensitive/new_relic_metric_path: /metric/v1
+/bfd/${env}/pipeline/nonsensitive/shared/bfd_pipeline_dir: /bluebutton-data-pipeline
+/bfd/${env}/pipeline/nonsensitive/shared/data_pipeline_dir: /bluebutton-data-pipeline
+/bfd/${env}/pipeline/nonsensitive/shared/data_pipeline_new_relic_metric_host: https://gov-metric-api.newrelic.com
+/bfd/${env}/pipeline/nonsensitive/shared/data_pipeline_new_relic_metric_path: /metric/v1
+/bfd/${env}/pipeline/nonsensitive/shared/data_pipeline_user: bb-etl
+/bfd/${env}/pipeline/nonsensitive/shared/data_pipeline_micrometer_cw_enabled: false
+/bfd/${env}/pipeline/nonsensitive/shared/data_pipeline_micrometer_cw_namespace: bfd-${env}/bfd-pipeline
+/bfd/${env}/pipeline/nonsensitive/shared/data_pipeline_micrometer_cw_interval: PT1M
+/bfd/${env}/pipeline/nonsensitive/ccw/data_pipeline_rda_job_enabled: false # RDA Jobs are disabled by default on CCW Pipelines
+/bfd/${env}/pipeline/nonsensitive/ccw/data_pipeline_ccw_rif_job_enabled: false
+/bfd/${env}/pipeline/nonsensitive/ccw/data_pipeline_idempotency_required: true
+/bfd/${env}/pipeline/nonsensitive/ccw/instance_type: c6a.4xlarge
+/bfd/${env}/pipeline/nonsensitive/ccw/rif_job_batch_size: 7
+/bfd/${env}/pipeline/nonsensitive/ccw/rif_job_queue_size_multiple: 5
+/bfd/${env}/pipeline/nonsensitive/ccw/rif_thread_multiple: 3
+/bfd/${env}/pipeline/nonsensitive/ccw/rif_job_batch_size_claims: 100
+/bfd/${env}/pipeline/nonsensitive/ccw/rif_job_queue_size_multiple_claims: 10
+/bfd/${env}/pipeline/nonsensitive/ccw/rif_thread_multiple_claims: 25
+/bfd/${env}/pipeline/nonsensitive/ccw/slis_repeater_lambda_invoke_rate: 15 minutes
+/bfd/${env}/pipeline/nonsensitive/rda/grpc/inprocess_server/mode: UNDEFINED
+/bfd/${env}/pipeline/nonsensitive/rda/grpc/inprocess_server/s3/region: us-east-1
+/bfd/${env}/pipeline/nonsensitive/rda/grpc/inprocess_server/s3/bucket: UNDEFINED
+/bfd/${env}/pipeline/nonsensitive/rda/grpc/inprocess_server/s3/dir: UNDEFINED
```

Comment:
Both of these values were originally left blank, which yields a string value of "null." Instead, we'd rather not define these parameters at all for ephemeral environments.